vmm/src/api/openapi/cloud-hypervisor.yaml

```diff
@@ -1110,6 +1128,12 @@ components:
         destination_url:
           type: string
 
+    VmCoredumpData:
+      type: object
+      properties:
+        filepath:
```

Comment:
I would name this `destination_uri` so it's clear it's a URL.

Reply:
Okay, thanks. So would you do this when merge or should I rebase another commit? @rbradford

Reply:
Please fix it directly on this PR, by updating the commit where you introduced the code. (So yes please rebase)

Reply:
Ok, I'll rebase immediately.

Reply:
The parameter and member of structure have been renamed as suggestions. @rbradford @sboeuf